package.json

```diff
@@ -60,7 +60,7 @@
     "@fluentui/style-utilities": "8.9.6",
     "@fluentui/theme": "2.6.25",
     "@fluentui/web-components": "^2.5.12",
-    "@lexical/react": "^0.11.3",
+    "@lexical/react": "^0.12.4",
```

Comment:
Doing some validation tests locally on this change.

Reply:
Found some weird behavior/bugs, but it all seems to be present in v0.11.3 (i.e., not a regression), so marking validation as complete.